docs/state-store-overview.md

```diff
@@ -2,21 +2,29 @@
 
 - [An Overview of RisingWave State Store](#an-overview-of-risingwave-state-store)
   - [Overview](#overview)
-  - [Architecture](#architecture)
+  - [Cell-based Relational Table](#Relational-table)
+    - [Write Path](#relational-table-write-path)
+    - [Read Path](#relational-table-read-path)
```

Comment:
Not sure whether the label of the parent node is necessary. Maybe "read-path" or "write-path" would be fine

Reply:
This is just the same as the original.

Reply:
Oh, I got the reason: we have multiple sections named "write-path" and "read-path". So this is necessary.